pkg/cloud/services/ec2/vpc_test.go

```diff
@@ -142,9 +142,10 @@ func TestReconcileVPC(t *testing.T) {
 					}, nil)
 
 				m.DescribeVpcAttribute(gomock.AssignableToTypeOf(&ec2.DescribeVpcAttributeInput{})).
-					DoAndReturn(describeVpcAttributeFalse).AnyTimes()
```

Comment:
I can't comment on line 117 above - do you need to remove `CidrBlock` to match https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/916/files#diff-0133c773261155a5c13f644de72eb616L117?